config/config.json

```diff
@@ -5,7 +5,7 @@
     },
     "http_server": {
         "base_uri": "http://localhost:8080/v2/management",
```

Comment:
remove or fix port in base_uri. It works without it

Reply:
Done